ansible/playbooks/roles/repository/tasks/RedHat/install-packages.yml

```diff
@@ -1,28 +1,26 @@
 ---
-- include_vars:
-    file: roles/common/vars/main.yml
-    name: common_vars
+- name: Get information on installed packages
+  package_facts:
+    manager: rpm
+  when: ansible_facts.packages is undefined
 
-# Fix to solve issues with expiring certificates on Azure
+# Update RHUI client certificate
 # https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-rhui#update-expired-rhui-client-certificate-on-a-vm
-# TODO: Needs to be updated for RHEL 8
-- name: Update repository certificates
+- name: Update RHUI client package
   yum:
-    enablerepo: "*microsoft*"
-    disablerepo: "*"
-    state: "latest"
+    enablerepo: 'rhui-microsoft-azure*'
```

Comment:
Because IMO `*microsoft*` pattern is too wide and thus not informative. When I see `rhui-microsoft-azure` then I know that this is for RHUI (Red Hat Update Infrastructure). The new pattern works fine 😉